airbyte-integrations/connectors/source-zendesk-support/Dockerfile

```
@@ -25,5 +25,5 @@ COPY source_zendesk_support ./source_zendesk_support
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.3.1
+LABEL io.airbyte.version=0.4.0
```

I think this is just a minor version change, since it looks like it is backwards compatible.

```suggestion
LABEL io.airbyte.version=0.3.2
```

---

New streams are minor update as described in our docs https://docs.airbyte.com/contributing-to-airbyte/#examples
The version `0.4.0` is correct.

---

Not sure what I was thinking yesterday 🤦 Thank you